spec/lib/resolve.spec.js

```diff
@@ -5,60 +5,58 @@ var resolve = require("../../lib/resolve");
 var baseDir = path.resolve(__dirname, "../..");
 
 
-describe("lib/", function () {
-  describe("resolve", function () {
```

Comment:
I removed this inner describe and made the top level `lib/resolve` to make it similar to the other tests. The rest is just whitespace.